docs/plugins/create-a-plugin.md

```diff
@@ -39,3 +39,11 @@ yarn workspace @backstage/plugin-my-plugin start # Also supports --check
 This method of serving the plugin provides quicker iteration speed and a faster
 startup and hot reloads. It is only meant for local development, and the setup
 for it can be found inside the plugin's `dev/` directory.
+
+### Other Plugin Types
+
+There are other plugin types that you can chose from. To be able to select the type
+when you create a new plugin just run: `yarn new`. You'll then be asked what type of
+plugin you wish to create like this:
+
+![List of available plugin types to pick from](../assets/plugins/create-plugin_types.png)
```

Comment:
Thanks for uploading!! There is a little, visual bug where a black box hides the "_backend_" word in the first row. Not crucial, but could be updated quickly :)

Reply:
Good catch, completely missed that! I've corrected it so should be better now 👍